src/pages/Calculator.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import Display from '../components/display';
+import CalculatorButton from '../components/calculatorButton';
+import calculate from '../logic/calculate';
+import '../styles/calculator.css';
+
+const Calculator = () => {
+  const [obj, setObj] = useState({ total: null, next: null, operation: null });
+  const clickHandler = (button) => {
+    const newState = calculate(obj, button);
+    setObj(newState);
```

Comment:
- [optional] everything works fine. you can design your nave bar to be different, also you can go further to make the current user page link different or highlighted. great job
- [optional] it will be better if you centralize your calculator app on the page.